docs/06-web-development.Rmd

```diff
@@ -101,7 +101,70 @@ There are much better tutorials on HTML than this section, such as those offered
 
 ## CSS
 
-CSS is a lot of fun to play with, but it can also easily steal your time.
+The Cascading Stylesheets (CSS) language is used to describe the look and formatting of documents written in HTML or XML. CSS is responsible for the visual style of your site. CSS is a lot of fun to play with, but it can also easily steal your time.
+
+In the Hugo framework (https://gohugo.io/tutorials/creating-a-new-theme/), CSS is one of the major “non-content” files that shapes the look and feel of your site (the others are images, JavaScript, and Hugo templates). What does the ["look and feel"](https://en.wikipedia.org/w/index.php?title=Look_and_feel&oldid=731052704) of a site mean? “Look” generally refers to static style components including, but not limited to:
+
+* Color palettes
+* Images
+* Layouts/margins
+* Fonts
+
+Whereas “feel” relates to dynamic components that the user interacts with like:
+
+* Dropdown menus
+* Buttons
+* Forms
+
+There are 3 ways to define styles. The first is inline with HTML. For example, this code:
+
+
+```html
+<text style="color:red">Marco!</text>
+Polo!
+```
+Would produce text that looks like this:
+<text style="color:red">Marco!</text>
+Polo!
```

Comment:
Please remove the example output since it won't work in PDF. 

Is `<text>` a valid HTML tag?